tikv/gc.go

```diff
@@ -81,8 +81,9 @@ func WithConcurrency(concurrency int) GCOpt {
 }
 
 func (s *KVStore) resolveLocks(ctx context.Context, safePoint uint64, concurrency int) error {
+	lockResolver := &BaseLockResolver{store: s}
 	handler := func(ctx context.Context, r kv.KeyRange) (rangetask.TaskStat, error) {
-		return s.resolveLocksForRange(ctx, safePoint, r.StartKey, r.EndKey)
+		return ResolveLocksForRange(ctx, "GC", lockResolver, safePoint, r.StartKey, r.EndKey)
```

Comment:
If the second parameter is named `uuid`, it doesn't make sense to pass `"GC"` to it. As there isn't a GCWorker here, consider generating one.
Actually I think we don't need to keep `uuid` anyway. For example, we can call it `identifier`, and pass it `"gc-client-go-api"` here, and pass `"gcworker-{uuid}"` from TiDB.

Reply:
Got it